src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -7,7 +7,25 @@
         {% set group_class = ' form-control is-invalid' %}
         {% set valid = true %}
     {% endif %}
-    {{- parent() -}}
+    {% set prepend = not (money_pattern starts with '{{') %}
+    {% set append = not (money_pattern ends with '}}') %}
+    {% if prepend or append %}
```

Comment:
it feel like the whole new block here misses a few `-` to remove leading/trailing whitespaces